salt/modules/boto_elbv2.py

```diff
@@ -55,6 +55,8 @@
     # pylint: enable=unused-import
 
     # TODO Version check using salt.utils.versions
+    import boto3
```

Comment:
These imports will need to be moved up into the `unused-import` section with the `import salt.utils.boto3` line:

https://jenkins.saltstack.com/job/PR/job/salt-pr-lint-n/15590/violations/file/salt/modules/boto_elbv2.py/